tests/App/DataFixtures/MongoDB/BookFixtures.php

```diff
@@ -29,9 +29,6 @@ public function load(ObjectManager $manager): void
         $manager->flush();
     }
 
-    /**
-     * @phpstan-return class-string[]
```

Comment:
why ?

Reply:
there is a more specific return type in the interface: https://github.com/doctrine/data-fixtures/blob/4165e20b8f89827aa5a71229541382a276ced8f9/lib/Doctrine/Common/DataFixtures/DependentFixtureInterface.php#L16